### DISCUSSION TITLE
docker-compose project archived

### DISCUSSION THREAD

**Discussion by fr33ek** [api-platform/api-platform#1839] [Q&A]
Hello!

Making my first steps with api-platform. As a first step I want to deploy my app using `docker-compose`. Official [doc](https://api-platform.com/docs/deployment/docker-compose/) leads to https://github.com/api-platform/docker-compose-prod which is deprecated. So what is a **preferred** way of deploying user `docker-compose`?

Thanks in advance!

**Comment by andrewmy** [comment]
There's a `docker-compose.yml` file included right into this repository.
The doc should be fixed though.
